apps/web-giddh/src/app/app.routes.ts

```diff
@@ -97,7 +97,7 @@ export const ROUTES: Routes = [
             { path: 'permissions', loadChildren: './permissions/permission.module#PermissionModule', canActivate: [NeedsAuthorization] },
             { path: 'settings', loadChildren: './settings/settings.module#SettingsModule', canActivate: [NeedsAuthorization] },
             { path: 'manufacturing', loadChildren: './manufacturing/manufacturing.module#ManufacturingModule', canActivate: [NeedsAuthorization] },
-            { path: 'accounting-voucher', loadChildren: './accounting/accounting.module#AccountingModule' },
+            // { path: 'accounting-voucher', loadChildren: './accounting/accounting.module#AccountingModule' },
```

Comment:
will this commented code reuse in future?

Reply:
It's JV code commented for some time as new JV module is under development.